examples/next-liveness/theme.ts

```diff
@@ -0,0 +1,108 @@
+import { defaultTheme, Theme } from '@aws-amplify/ui-react';
+import { flipPalette } from 'utils';
+
+export const theme: Theme = {
+  name: 'listings-theme',
+  breakpoints: {
+    values: {
+      large: 50,
+    },
+  },
+  tokens: {
+    colors: {
+      brand: {
+        // primary: usePalette('blue'),
+        // secondary: usePalette('red')
+      },
+      border: {
+        // Dark borders
+        primary: { value: '{colors.neutral.100.value}' },
+        secondary: { value: '{colors.neutral.90.value}' },
+        tertiary: { value: '{colors.neutral.60.value}' },
+      },
+    },
+    radii: {
+      // Rounded
+      // small:  { value: '1rem' },
+      // medium: { value: '2rem' },
+      // large:  { value: '3rem' },
+
+      // Squared up
+      small: { value: '0' },
+      medium: { value: '0' },
+      large: { value: '0' },
+      xl: { value: '0' },
+      xxl: { value: '0' },
+    },
+    space: {
+      // extra space
+      // xxs:    { value: '0.5rem' },
+      // xs:     { value: '0.75rem' },
+      // small:  { value: '1rem' },
+      // medium: { value: '2rem' },
+      // large:  { value: '3rem' }
```

Comment:
Do we need to keep this code for any reason? Same q for other commented out values.

Reply:
probably just missed removing it, will remove